CI failed on LLVM 14

The LLVM 14 Release test keeps failing, both on Github Action of my forked master branch and on my ubuntu machine. Because the docker image is cached, all actions on this repo are not affected yet.

Error message: 
```bash
[ 93%] Building C object src/cc/CMakeFiles/bcc-shared.dir/bcc_proc.c.o
make[3]: *** No rule to make target '/usr/lib/llvm-14/lib/libPolly.a', needed by 'src/cc/libbcc.so.EAD-HASH-NOTFOUND'.  Stop.
make[3]: *** Waiting for unfinished jobs....
[ 96%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/common.cc.o
make[2]: *** [CMakeFiles/Makefile2:430: src/cc/CMakeFiles/bcc-shared.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:437: src/cc/CMakeFiles/bcc-shared.dir/rule] Error 2
make: *** [Makefile:244: bcc-shared] Error 2
```

It seems related to this [issue](https://discourse.llvm.org/t/ubuntu-packages-with-llvm-14-then-15-suddenly-looking-for-libpolly-a/67655), I tried downgrade the llvm-14 to a earlier version, the build passed.

### What reproduces the bug?
Run LLVM-14 Release CI tests locally.

--- a/docker/Dockerfile.focal
+++ b/docker/Dockerfile.focal
@@ -57,6 +57,11 @@ RUN /sbin/bpftool &>/dev/null || (rm -f /sbin/bpftool && ln -s /usr/lib/linux-to
 
 RUN if [ "$LLVM_VERSION" -ge 13 ] ; then apt-get install -y libmlir-${LLVM_VERSION}-dev ; fi
 
+RUN if [ "$LLVM_VERSION" -ge 14 ] ; then \
+      apt download libpolly-${LLVM_VERSION}-dev && \
+      dpkg --force-all -i libpolly-${LLVM_VERSION}-dev*.deb ; \
+    fi
+
 RUN ln -s /usr/bin/python3 /usr/bin/python
 COPY build.sh /build.sh
 RUN chmod 755 /build.sh

